src/Form/index.tsx

```diff
@@ -1172,20 +1121,15 @@ function Form({
       const action = actions[i];
       const type = action.type;
 
-      if (
-        [ACTION_ADD_REPEATED_ROW, ACTION_REMOVE_REPEATED_ROW].includes(type)
-      ) {
-        let data: any;
-        if (type === ACTION_ADD_REPEATED_ROW) data = addRepeatedRow();
-        else data = removeRepeatedRow(element.repeat);
-        if (data && data.fieldKeys.length > 0) {
-          fieldOnChange({
-            fieldIds: data.fieldIDs,
-            fieldKeys: data.fieldKeys,
-            elementRepeatIndex: element.repeat
-          })();
-        }
-      } else if (type === ACTION_TRIGGER_PLAID) {
+      await runUserCallback(onAction, () => ({
+        trigger,
+        action: type
```

Comment:
This adds 'action' to the callback context but it is not in ContextOnAction (Form.ts).  It should be, no?

Reply:
yes, thank you for catching